--- sundials-2.6.2/examples/idas/serial/idasSlCrank_FSA_dns.c	2015-07-30 17:08:56.000000000 -0700
+++ patch-files/idasSlCrank_FSA_dns.c	2016-02-02 12:39:25.864488992 -0800
@@ -74,7 +74,7 @@
 static int rhsQ(realtype t, N_Vector yy, N_Vector yp, N_Vector qdot, void *user_data);
 
 static int rhsQS(int Ns, realtype t, N_Vector yy, N_Vector yp, 
-                 N_Vector *yyS, N_Vector *ypS, N_Vector rrQ, N_Vector *rhsQS,
+                 N_Vector *yyS, N_Vector *ypS, N_Vector rrQ, N_Vector *temprhsQS,
                  void *user_data,  N_Vector yytmp, N_Vector yptmp, N_Vector tmpQS);
 
 
@@ -464,7 +464,7 @@
 }
 
 static int rhsQS(int Ns, realtype t, N_Vector yy, N_Vector yp, 
-                 N_Vector *yyS, N_Vector *ypS, N_Vector rrQ, N_Vector *rhsQS,
+                 N_Vector *yyS, N_Vector *ypS, N_Vector rrQ, N_Vector *temprhsQS,
                  void *user_data,  N_Vector yytmp, N_Vector yptmp, N_Vector tmpQS)
 {
   realtype v1, v2, v3;
@@ -488,13 +488,13 @@
   s2 = Ith(yyS[0],5);
   s3 = Ith(yyS[0],6);
 
-  Ith(rhsQS[0], 1) = J1*v1*s1 + m2*v2*s2 + J2*v3*s3;
+  Ith(temprhsQS[0], 1) = J1*v1*s1 + m2*v2*s2 + J2*v3*s3;
 
   s1 = Ith(yyS[1],4);
   s2 = Ith(yyS[1],5);
   s3 = Ith(yyS[1],6);
 
-  Ith(rhsQS[1], 1) = J1*v1*s1 + m2*v2*s2 + J2*v3*s3;
+  Ith(temprhsQS[1], 1) = J1*v1*s1 + m2*v2*s2 + J2*v3*s3;
 
   return(0);
 }
