--- sundials-2.6.2/examples/idas/serial/idasRoberts_ASAi_dns.c	2015-07-30 17:08:55.000000000 -0700
+++ patch-files/idasRoberts_ASAi_dns.c	2016-02-02 12:39:25.864488992 -0800
@@ -122,7 +122,7 @@
 static int JacB(long int NeqB, realtype tt, realtype cjB,
                 N_Vector yy, N_Vector yp,
                 N_Vector yyB, N_Vector ypB, N_Vector rrB, 
-                DlsMat JacB, void *user_data,
+                DlsMat tempJacB, void *user_data,
                 N_Vector tmp1B, N_Vector tmp2B, N_Vector tmp3B);
 
 
