--- lapack-3.8.0/make.inc.example	2017-11-12 20:15:54.000000000 -0800
+++ patch-files/make.inc.example	2018-11-20 12:12:08.119482878 -0800
@@ -8,8 +8,8 @@
 
 #  CC is the C compiler, normally invoked with options CFLAGS.
 #
-CC     = gcc
-CFLAGS = -O3
+CC     = ROLL_CC
+CFLAGS =
 
 #  Modify the FORTRAN and OPTS definitions to refer to the compiler
 #  and desired compiler options for your machine.  NOOPT refers to
@@ -19,15 +19,15 @@
 #  and handle these quantities appropriately. As a consequence, one
 #  should not compile LAPACK with flags such as -ffpe-trap=overflow.
 #
-FORTRAN = gfortran
-OPTS    = -O2 -frecursive
+FORTRAN = ROLL_FORTRAN
+OPTS    = 
 DRVOPTS = $(OPTS)
-NOOPT   = -O0 -frecursive
+NOOPT   = 
 
 #  Define LOADER and LOADOPTS to refer to the loader and desired
 #  load options for your machine.
 #
-LOADER   = gfortran
+LOADER   = ROLL_FORTRAN
 LOADOPTS =
 
 #  The archiver and the flag(s) to use when building an archive
@@ -47,7 +47,7 @@
 #TIMER = EXT_ETIME_
 #  For gfortran compiler:  SECOND and DSECND will use a call to the
 #  INTERNAL FUNCTION ETIME
-TIMER = INT_ETIME
+TIMER = INT_CPU_TIME
 #  If your Fortran compiler does not provide etime (like Nag Fortran
 #  Compiler, etc...) SECOND and DSECND will use a call to the
 #  INTERNAL FUNCTION CPU_TIME
