--- sana_aux_par.F	2017-10-23 16:01:24.906213006 -0700
+++ patch-files/sana_aux_par.F	2017-10-23 16:01:54.807213942 -0700
@@ -1390,7 +1390,7 @@
       SUBROUTINE SMUMPS_BUILD_SCOTCH_GRAPH(id, FIRST, LAST, IPE,
      &     PE, WORK)
       IMPLICIT NONE
-      INCLUDE 'mpif.h'
+C     INCLUDE 'mpif.h'
       TYPE(SMUMPS_STRUC)      :: id
       INTEGER, POINTER        :: FIRST(:), LAST(:), IPE(:), PE(:),
      &     WORK(:)
@@ -1587,7 +1587,7 @@
       SUBROUTINE SMUMPS_BUILD_LOC_GRAPH(id, ord, GSIZE, IPE, PE, LENG,
      &     I_HALO_MAP, top_graph, WORK)
       IMPLICIT NONE
-      INCLUDE 'mpif.h'
+C     INCLUDE 'mpif.h'
       TYPE(SMUMPS_STRUC)   :: id
       TYPE(ORD_TYPE)       :: ord
       TYPE(GRAPH_TYPE)     :: top_graph
@@ -1863,7 +1863,7 @@
       SUBROUTINE SMUMPS_SEND_BUF(APNT, PROC, NPROCS, BUFSIZE, IPE, PE,
      &     LENG, RCVBUF, MSGCNT, SNDCNT, COMM)
       IMPLICIT NONE
-      INCLUDE 'mpif.h'
+C     INCLUDE 'mpif.h'
       INTEGER                 :: NPROCS, PROC, COMM
       TYPE(ARRPNT)            :: APNT(:)
       INTEGER                 :: BUFSIZE
